### PR TITLE
Generate waveforms at unequally sampled frequencies

### PR DESCRIPTION
**Pull request by ahnitz**
At code to get waveforms at discrete sets of frequency points and to attempt reconstruct it. Warning, that the reconstruction code does not work in all cases at this point.

### CODE REVIEW

File: pycbc/waveform/compress.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    -------
+    time: float
+        Time from flow untill the end of the waveform
+    """
+    m = m1 + m2
+    msun = m * lal.MTSUN_SI
+    t =  5.0 / 256.0 * m * m * msun / (m1 * m2) / (numpy.pi * msun * flow) **  (8.0 / 3.0)
+
+    # fudge factoriness
+    return .022 if t < 0 else (t + .02) * 1.1
```
Comment (cdcapano) [inline pycbc/waveform/compress.py]: Can you make these fudge factors optional arguments to the function?

File: pycbc/waveform/compress.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        size of frequency step
+    flow: float
+        the lower frequency cutoff of the waveform
+    flen: float, optional
+        length of output waveform (either provide this or out)
+    out: pycbc.types.FrequencySeries, optional
+        output memory (either provide this or flen)
+
+    Returns
+    -------
```
Comment (cdcapano) [inline pycbc/waveform/compress.py]: What happened to the rest of the documentation?

**Comment by ahnitz** [comment]
Collin, I've removed all the bits that will need further thought. Please re-examine.

**Comment by cdcapano** [comment]
Looks good; just one issue. Add ``get_fd_waveform_sequence`` to waveform.py's ``__add__``.

**Comment by ahnitz** [comment]
@cdcapano Ok, done

**Pull request merged**